ProjectsNetwork/Startup.cs

```diff
@@ -46,14 +46,14 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<IProjectSkillRepository, ProjectSkillRepository>();
             services.AddScoped<IInterestedInProjectRepository, InterestedInProjectRepository>();
             services.AddScoped<IApplicationUserRepository, ApplicationUserRepository>();
-            services.AddSingleton<IEmailSender, EmailSender>();
 
 
             services.AddScoped<ISkillsService, SkillsService>();
             services.AddScoped<IProjectsService, ProjectsService>();
             services.AddScoped<IApplicationUserService, ApplicationUserService>();
-            
 
+            services.AddTransient<IEmailSender, EmailSender>();
+            services.Configure<AuthMessageSenderOptions>(Configuration);
```

Comment:
I don't see any user secrets being pushed to Github repo. How is it supposed to get the values if the phisical files are not in the repo? It won't work on anybody's computer if pulled.